lib/get-category-path.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var path = require('path'),
+  utility = require('./utility');
+
+
+/**
+ * Determines the category directory structure from the pattern's data
+ *
+ * @param {Object} patternObject  individual pattern's data (from /pattern-name/pattern.yml)
+ * @param {Object} options  pattern-importer options
+ *
+ * @return {String} categoryPath  internal path url to category
+ *
+ * @requires ./lib/utility
+ * @requires ./lib/category-name-converter
+ */
+exports.getCategoryPath = function (patternObject, options) {
```

Comment:
not sure what you mean...are you talking about the spacing between parenthesis & curly brackets?